browser/src/control/Control.Notebookbar.js

```diff
@@ -165,8 +165,10 @@ L.Control.Notebookbar = L.Control.extend({
 			this._showNotebookbar = true;
 			this.showTabs();
 			$('.main-nav').removeClass('readonly');
+			$('#toolbar-down').removeClass('readonly');
```

Comment:
Please try to avoid modifying other components in unrelated files. Then we have clear separation between them.
This should be done in the `Status bar` code or in the `UIManager` which shows/hides components (if it depends on the used view mode).

Statusbar:  https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.StatusBar.js#L445

UIManager: https://github.com/CollaboraOnline/online/blob/master/browser/src/control/Control.UIManager.js#L629

Reply:
done, added to statusbar thanks